srcs/parser/globbing/globbing_run.c

```diff
@@ -56,6 +56,23 @@ static char	*s_join_free(char *s1, char *s2)
 	return (ret);
 }
 
+typedef struct s_mylist
+{
+	struct s_mylist		*next;
+	char				*content;
+}					t_mylist;
+
+void globbing_bracket(t_mylist **list, char *input)
```

Comment:
This is the function to fill.

Basic example to test:

``` bash
mkdir -p tmp/a tmp/b
touch tmp/a/aa tmp/a/aaa
touch tmp/b/bb tmp/b/bbb
touch tmp/Z
```

with the command

``` bash
bash-3.2$ ls tmp/*/*[a..b]
tmp/a/aa	tmp/a/aaa	tmp/b/bb	tmp/b/bbb
```

@gabkk